pyweather.py

```diff
@@ -1909,7 +1956,156 @@ def radar_clearImages():
     summary_forecastIterations = summary_forecastIterations + 1
     if summary_forecastIterations == 5:
         break
+
 print("")
+
+logger.debug("yesterdayJSON loaded with: %s" % yesterdayJSON)
+yesterday_json = json.loads(yesterdayJSON.text)
+if jsonVerbosity == True:
+    logger.debug("yesterday_json: %s" % yesterday_json)
+else:
+    logger.debug("Loaded 1 JSON.")
+yesterday_date = yesterday_json['history']['date']['pretty']
+print(Fore.YELLOW + "Here's yesterday's weather for " + Fore.CYAN +
+      str(location) + Fore.YELLOW + " on "
+      + Fore.CYAN + yesterday_date)
+logger.debug("yesterday_date: %s" % yesterday_date)
+for data in yesterday_json['history']['dailysummary']:

```

Comment:
You'll want to parse this data before we start to show weather conditions, and do this in a conditional for if showing yesterday's weather is true.

Reply:
Done.